bluebutton-data-model-rif/src/main/resources/db/migration/V15__Add_Inner_Join_Relationships.sql

```diff
@@ -0,0 +1,25 @@
+/*
+ * Alters the Beneficiaries History table to add foreign key to Beneficiary 
+ * 
+ * See:
+ * * https://jira.cms.gov/browse/BLUEBUTTON-620
+ */
+
+
+alter table "Beneficiaries" add column "beneficiaryHistories" varchar(255)
+alter table "Beneficiaries" add column "medicareBeneficiaryIdHistories" varchar(255)
```

Comment:
I don't think these actual columns are needed. Instead, JPA ought to be creating them as dynamic fields just in the entity classes, right?